src/back/spv/mod.rs

```diff
@@ -291,6 +291,7 @@ enum LocalType {
     PointerToBindingArray {
```

Comment:
Pre-existing, but suggest:

```suggestion
    /// Equivalent to a [`LocalType::Pointer`] whose `base` is a Naga IR [`BindingArray`]. SPIR-V
    /// permits duplicated `OpTypePointer` ids, so it's fine to have two different [`LocalType`]
    /// representations for pointer types.
    ///
    /// [`BindingArray`]: crate::TypeInner::BindingArray
    PointerToBindingArray {
```